composer.json

```diff
@@ -44,6 +44,7 @@
         "symfony/yaml": "~2.3",
         "symfony/security-acl": "~2.4",
         "sonata-project/intl-bundle": "~2.1",
+        "sllh/php-cs-fixer-styleci-bridge": "~1.0",
```

Comment:
Also, note that php-cs-fixer 2.x is already significantly different from 1.x even before my changes.